pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -854,6 +854,14 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, podStatus *kubecontaine
 		klog.V(4).InfoS("Creating container in pod", "containerType", typeName, "container", spec.container, "pod", klog.KObj(pod))
 		// NOTE (aramase) podIPs are populated for single stack and dual stack clusters. Send only podIPs.
 		if msg, err := m.startContainer(podSandboxID, podSandboxConfig, spec, pod, podStatus, pullSecrets, podIP, podIPs); err != nil {
+			// startContainer can return an error from CNI, CSI, or
+			// CRI if the Pod has been deleted while the POD is
+			// being created. If the pod has been deleted then it's
+			// not a real error.
+			if m.podStateProvider.IsPodDeleted(pod.UID) {
+				klog.V(4).InfoS("Pod was deleted and startContainer failed to be created", "pod", klog.KObj(pod), "podUID", pod.UID)
```

Comment:
Is `V(4)` too high? In most cases it will not be enabled and if this introduces some regression, it will be impossible to troubleshoot

Reply:
V(3) preferred?